app/views/chargeback/_cb_tier_edit_values.html.haml

```diff
@@ -11,5 +12,9 @@
   = text_field_tag("fixed_rate_#{detail_index}_#{row_within_rate}", tier[:fixed_rate],
     :maxlength => MAX_NAME_LEN, 'data-miq_observe' => {:interval => '.5', :url => url}.to_json)
 %td{:align => 'right'}
-  = text_field_tag("variable_rate_#{detail_index}_#{row_within_rate}", tier[:variable_rate],
-    :maxlength => MAX_NAME_LEN, 'data-miq_observe' => {:interval => '.5', :url => url}.to_json)
+  - if detail['group'] == 'fixed' && (!tier[:variable_rate].present? || tier[:variable_rate].zero?)
```

Comment:
Not a big deal, but I would perhaps change `!tier[:variable_rate].present?` to `tier[:variable_rate].blank?`